fractal-puppy/Engine
# Audio files added as Resource Audio

Audio files now are copied to "Library/Audios" and loaded only once. You can view the import configuration using panel browser to load them as "streamed", as well as delete or rename them.

To test (1):
- Load ResourceAudioTest.sc3ne
- Open the ResourceManager tool.
- Load one song on AudioSource GO using the component.
- Check that the audio resource number of references for the file has increased.
- Load the same song on the other AudioSource GO and check that now the number is 2.

To test (2):
- Using the PanelBrowser go to Audio.
- Left click on any audio file.
- Select "Import Configuration"
- Check the checkbox and accept.
- Close the engine.
- Open the engine.
- Using the PanelBrowser go to Audio and left click on the same audio file.
- Select "Import Configuration" and see if the checkbox is checked.

To test (3):
- Using the PanelBrowser go to Audio.
- Left click on any audio file.
- Select "Rename"
- Give it a new name
- Open the ResourceManager tool.
- Check that the resource audio has the new name.

To test (4):
- Using the PanelBrowser go to Audio.
- Left click on any audio file.
- Select "Delete"
- Open the ResourceManager tool.
- Check that the resource audio has been deleted.
